.github/CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+# Repository owner
+* @jonzeolla
+
+# GitHub automation oversight
+.github/* @SeisoLLC/eyes
```

Comment:
```suggestion
.github/** @SeisoLLC/eyes
```

Reply:
Good catch ty, forgot that didn't make it into upstream yet